### PR TITLE
New action and fuzzy logic

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Descriptions:
1. Provided check box for fuzzy search in action "Filter JSON by Key Value"
2. Implemented new action to search keyword in text as "Search Keyword In Text", which by default support Fuzzy Match

UTC:
1. Verified that on checking Fuzzy Match checkbox, fuzzy filter happens.

### CODE REVIEW

File: fuzzy-search/info.json [hunk 1]
```diff
@@ -12,42 +12,33 @@
             "operation": "filter_json_by_key_value",
             "title": "Filter JSON by Key Value",
             "annotation": "filter_json_by_key_value",
-            "description": "Filters JSON as per provided key value pair.",
+            "description": "Filters JSON as per provided key value pair in JSON.",
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: Suggestion: Filters input JSON as per provided key value pair as JSON object

File: fuzzy-search/info.json [hunk 2]
```diff
@@ -12,42 +12,33 @@
...
-                    "title": "Target Key",
-                    "type": "text",
-                    "name": "targetKey",
-                    "required": true,
-                    "visible": true,
-                    "editable": true,
-                    "description": "Specify target key for data filtering.",
-                    "tooltip": "Specify target key for data filtering."
+                    "value": null,
+                    "tooltip": "Specify list of JSON or JSON to filter data.",
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: Suggestion: Specify a input JSON or JSON list

File: fuzzy-search/info.json [hunk 3]
```diff
@@ -12,42 +12,33 @@
...
                 {
-                    "title": "Target Key's Value",
-                    "type": "text",
+                    "title": "Target Key Value",
+                    "type": "object",
                     "name": "targetKeyValue",
                     "required": true,
                     "visible": true,
                     "editable": true,
                     "value": null,
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: default value can be added here

File: fuzzy-search/info.json [hunk 4]
```diff
@@ -12,42 +12,33 @@
...
-                    "description": "Specify target key's value for data filtering.",
-                    "tooltip": "Specify target key's value for data filtering."
+                    "tooltip": "Specify target key values pair for JSON data filtering.",
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: key value pairs and not key values

File: fuzzy-search/info.json [hunk 5]
```diff
@@ -57,12 +48,91 @@
...
+                },
+                {
+                    "title": "Fuzzy Match",
+                    "type": "checkbox",
+                    "name": "fuzzyMatch",
+                    "required": false,
+                    "visible": true,
+                    "editable": true,
+                    "value": true,
+                    "tooltip": "Check to filter data for fuzzy format.",
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: Suggestion: Select to filter data using fuzzy logic

File: fuzzy-search/info.json [hunk 6]
```diff
@@ -57,12 +48,91 @@
...
+                    }
                 }
             ],
             "output_schema": {},
             "open": false
+        },
+        {
+            "title": "Search Keyword In Text",
+            "operation": "search_keyword_in_text",
+            "description": "Searches specified fuzzy keyword in text",
```
Comment (cs-suyog-jinturkar) [inline fuzzy-search/info.json]: Suggestion: Fuzzy search specified keyword in an input text

File: fuzzy-search/info.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

File: fuzzy-search/info.json (hunk 2 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

File: fuzzy-search/info.json (hunk 3 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

File: fuzzy-search/info.json (hunk 4 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

File: fuzzy-search/info.json (hunk 5 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

File: fuzzy-search/info.json (hunk 6 above)
Comment (cs-ankit-gupta) [reply fuzzy-search/info.json]: Incorporated

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**